src/widget/wsearchlineedit.h

```diff
@@ -22,9 +23,13 @@ class WSearchLineEdit : public QComboBox, public WBaseWidget {
     static constexpr int kMaxDebouncingTimeoutMillis = 9999;
     static constexpr int kSaveTimeoutMillis = 5000;
     static constexpr int kMaxSearchEntries = 50;
+    static constexpr bool kEnableCompletionsByDefault = true;
+    static constexpr bool kEnableHistoryShortcutsByDefault = true;
```

Comment:
This is a function name, how about 
```suggestion
    static constexpr bool kDefaultSearchCompletionsEnabled = true;
    static constexpr bool kDefaultSearchHistoryShortcutsEnabled = true;
```

Reply:
[I preferred to remove](https://github.com/mixxxdj/mixxx/pull/10942#pullrequestreview-1189077067) the redundant 'search' string, so let's name these
`kCompletionsEnabledDefault`
`kHistoryShortcutsEnabledDefault`
?